src/commands/info/info.ts

```diff
@@ -0,0 +1,50 @@
+import { Command, container } from '@sapphire/framework';
+import { MessageEmbed } from 'discord.js';
+import {
+  CodeyCommand,
+  CodeyCommandResponseType,
+  SapphireMessageExecuteType,
+  SapphireMessageResponse
+} from '../../codeyCommand';
+
+const INFO_EMBED_COLOR = '#4287f5';
```

Comment:
is there a place where we store embed colours and such?

Reply:
The only embed in Codey right now is the Kick Embed, so that's stored in the embeds.ts under utils. I feel having the embed in the command itself is cleaner though.

Reply:
if we have more embeds in the future tho, i'd be nice to have consistent colours

Reply:
Fair enough. perhaps we could have some sort of "embed color" file or something like that.